solara/website/pages/__init__.py

```diff
@@ -115,6 +115,7 @@
     "documentation/examples/fullscreen/authorization": "/apps/authorization",
     "/examples/fullscreen/layout-demo": "/apps/layout-demo",
     "/documentation/examples/fullscreen/layout-demo": "/apps/layout-demo",
+    "/documentation/examples/fullscreen/layout_demo": "/apps/layout-demo",
```

Comment:
I think we should modify the entry above, not add a new one. The entry above never existed I think